docs/docs/modules/agents/agent_types/openai_assistant.mdx

```diff
@@ -0,0 +1,190 @@
+# OpenAI Assistant
+
+:::info
+The [OpenAI Assistant API](https://platform.openai.com/docs/assistants/overview) is still in beta. OpenAI does not recommend it for production use.
```

Comment:
Where do they say not recommended for prod?

Reply:
After checking again, I think that's something I hallucinated